ProjectCoimbra.UWP/Project.Coimbra/Pages/SongPage.xaml.cs

```diff
@@ -46,5 +51,38 @@ private async void NextButton_Click(object sender, RoutedEventArgs e)
 
             this.Next.IsEnabled = false;
         }
+
+        private async void Remove_Item(object sender, RoutedEventArgs e)
+        {
+            var res = ResourceLoader.GetForCurrentView();
+            this.ErrorBox.Text = string.Empty;
+
+            ContentDialog dialog = new ContentDialog
+            {
+                Title = res.GetString("SongPage/DeleteDialog/Title"),
+                Content = res.GetString("SongPage/DeleteDialog/Content"),
+                PrimaryButtonText = res.GetString("SongPage/DeleteDialog/Confirm"),
+                CloseButtonText = res.GetString("SongPage/DeleteDialog/Cancel"),
+            };
+
+            var result = await dialog.ShowAsync();
+
+            if (result == ContentDialogResult.Primary)
+            {
+                var item = (KeyValuePair<string, string>)((FrameworkElement)sender).DataContext;
+
+                try
+                {
+                    await SongPagesHelper.RemoveFile(null);
```

Comment:
Can you check if these async methods work with `.ConfigureAwait(false)` added? If so, that will make the code more efficient. If not, could you add `.ConfigureAwait(true)` so that we know this can't be used here.